modules/apps/object/object-test/src/testIntegration/java/com/liferay/object/service/test/ObjectEntryLocalServiceTest.java

```diff
@@ -621,6 +636,103 @@ public void testAddObjectEntry() throws Exception {
 				).build()));
 	}
 
+	@Test
+	public void testAddObjectEntryAsDraft() throws Exception {
+		_objectDefinition.setEnableObjectEntryDraft(true);
+
+		_objectDefinition =
+			_objectDefinitionLocalService.updateObjectDefinition(
+				_objectDefinition);
+
+		Map<String, Serializable> values1 =
```

Comment:
It would be `values`

Reply:
(Same comment as above) 

> I'm creating more than one variable here because I use them in a lambda expression in `AssertUtils.assertFailure` method and the variables used in lambda expressions cannot have its value changed.